### PR TITLE
Make EE ready

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Try to make this collection EE ready, with tests. Similar to https://github.com/ansible-collections/community.dns/pull/93.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
collection

### CODE REVIEW

File: tests/sanity/ignore-2.13.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tests/ee/roles/smoke/library/smoke_ipaddress.py shebang
```
Comment (briantist) [inline tests/sanity/ignore-2.13.txt:1]: why is this ignore necessary?

**Review by briantist — APPROVED** [review]
One question but otherwise looks good

File: tests/sanity/ignore-2.13.txt (hunk 1 above)
Comment (felixfontein) [reply tests/sanity/ignore-2.13.txt:1]: `ansible-test sanity` tries to force another shebang for non-module `.py` files, and does not recognize this to be a module. So I can either choose the wrong shebang - and fail the tests - (https://github.com/ansible-collections/community.hrobot/runs/6039348854?check_suite_focus=true; all other allowed shebangs also don't work), or I have to live with the sanity error (and ignore it).

Comment (felixfontein) [reply tests/sanity/ignore-2.13.txt:1]: These are in directories known to the sanity test: https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_util/controller/sanity/code-smell/shebang.py#L84-L92

**Comment by felixfontein** [comment]
@briantist thanks for reviewing this!

**Pull request merged**